services/supervisor/src/transactions.rs

```diff
@@ -25,7 +25,7 @@ use crate::{
 use super::{DeployConfirmation, DeployRequest, Error, Schema, StartService, Supervisor};
 
 /// Supervisor service transactions.
-#[exonum_service(crate = "crate")]
+#[exonum_service()]
 pub trait Transactions {
```

Comment:
nit: Can we name it `SupervisorInterface`, as `cryptocurrency-advanced` example does?